pyproject.toml

```
@@ -36,3 +36,6 @@ pre-commit = ">=4.0.1"
 [build-system]
 build-backend = "poetry.core.masonry.api"
 requires = ["poetry-core>=1.0.0"]
+
+[tool.poetry.extras]
+redis = ["redis"]
```

Using `too.poetry.extras` is deprecated, would you mind adding it via the modern way via `[project.optional-dependencies]`?

```suggestion

[project.optional-dependencies]
redis = ["redis>=5.2.0" ]
```

---

I updated this in my latest commit but what I changed it to doesn't actually work for me -- when I pull `signalbot` in as a dependency in my project, I get this error:

```
error: Failed to generate package metadata for `signalbot==0.13.0 @ directory+../signalbot`
  Caused by: Failed to parse metadata from built wheel
  Caused by: TOML parse error at line 40, column 2
   |
40 | [project.optional-dependencies]
   |  ^^^^^^^
`pyproject.toml` is using the `[project]` table, but the required `project.name` field is not set
```

And if I add that field it just complains about another, so it looks like it expects the file to be either fully converted over to `project` or entirely left on `tool.poetry`. Which would you prefer I do?

---

Let's keep it with `too.poetry.extras` and do bigger changes in a separate PR